.github/workflows/nightly.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Install cargo audit
         run: cargo install cargo-audit
       - name: Execute cargo audit
-        run: cargo audit
+        run: cargo audit --ignore RUSTSEC-2024-0006
```

Comment:
Can you add a small comment on top of the ignore command?

Reply:
Done